protected/humhub/modules/user/models/Group.php

```diff
@@ -472,4 +473,21 @@ public function getGroupSpaces()
     {
         return $this->hasMany(GroupSpace::class, ['group_id' => 'id']);
     }
+
+
+    /**
+     * Check if this Group can be deleted by current User
+     *
+     * @return bool
+     * @since 1.9
+     */
+    public function canDelete()
+    {
+        return Yii::$app->user->isAdmin() && !(
```

Comment:
We also have a `ManageGroups` permission, we may rather should check for this.

Reply:
Thanks, fixed in commit https://github.com/humhub/humhub/pull/4879/commits/774fab0ff7e84b69ba3fcef180d88232041ea862